ballerina/Ballerina.toml

```diff
@@ -15,5 +15,5 @@ graalvmCompatible = true
 [[platform.java21.dependency]]
 groupId = "io.ballerina.stdlib."
 artifactId = "mcp-native"
-version = "0.9.2"
-path = "../native/build/libs/mcp-native-0.9.2.jar"
+version = "0.9.3"
+path = "../native/build/libs/mcp-native-0.9.3-SNAPSHOT.jar"
```

Comment:
Is having snapshot versioning correct?

Reply:
yeah, when releasing this will get stripped anyway.